content/beginner/080_scaling/deploy_ca.files/cluster-autoscaler-autodiscover.yaml

```diff
@@ -138,15 +138,15 @@ spec:
     spec:
       serviceAccountName: cluster-autoscaler
       containers:
-        - image: k8s.gcr.io/cluster-autoscaler:v1.14.7
+        - image: k8s.gcr.io/cluster-autoscaler:v1.21.0
```

Comment:
The PR title says 1.20 but this container image is for 1.21. Please change to 1.20.

Reply:
New Commit submitted after modification based on the review comments.